src/vs/workbench/common/theme.ts

```diff
@@ -296,6 +296,15 @@ export const PANEL_ACTIVE_TITLE_BORDER = registerColor('panelTitle.activeBorder'
 	hcLight: '#B5200D'
 }, localize('panelActiveTitleBorder', "Border color for the active panel title. Panels are shown below the editor area and contain views like output and integrated terminal."));
 
+export const PANEL_TITLE_BADGE_BACKGROUND = registerColor('panelTitleBadge.background', {
```

Comment:
This should use `ACTIVITY_BAR_BADGE_BACKGROUND` as default. You might need to move the activity bar colors above the panel colors to do so.

Reply:
> You might need to move the activity bar colors above the panel colors to do so.

Yeah, I didn't know if it would be ok to move entire sections for this. Moved the `Panels` section below the `Activity Bar` now.